Aptoide/aptoide-client-v8
# [Feature/ASV-1057] AppNext Integration

**What does this PR do?**

   Integrates AppNext native ads to the Similar Ads in AppView. It currently replaces our ads there. 

**Database changed?**

   No

**Where should the reviewer start?**

- cm.aptoide.pt.ads.AppNextAdAnalytics
- cm.aptoide.pt.ads.model package

**How should this be manually tested?**

Open AppView, go to Similar Apps and check if the ad is retrieved from AppNext, aswell as verify that clicking launches the correct intent. Placement ID is in the ticket.

**What are the relevant tickets?**

  [ASV-1057](https://aptoide.atlassian.net/browse/ASV-1057)

**Notes:**

   Please pay close attention where the classes are placed. I don't really know the standards in use of where to put the classes. If they are not in the correct place (they probably won't be), please say.


**Code Review Checklist**
